tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,15 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+        it("should cache readFileSync on same file paths", () => {
+            const readFileSyncCount = sinon.spy(fs, "readFileSync");
+            const ignoredPaths = new IgnoredPaths({ ignore: true, cwd: getFixturePath() });
+
+            assert.isTrue(ignoredPaths.contains(getFixturePath(".eslintignore")));
+            sinon.assert.calledOnce(readFileSyncCount);
+            readFileSyncCount.restore();
```

Comment:
Can you put the spy logic in a `beforeEach`/`afterEach` block? As currently written, the spy won't be restored if there's an assertion failure, which could affect other tests and cause confusion.